packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_flutter_web.dart

```diff
@@ -327,4 +327,13 @@ class GoogleMapsPlugin extends GoogleMapsFlutterPlatform {
 
     return mapController.widget!;
   }
+
+  /// Populates [GoogleMapsFlutterInspectorPlatform.instance] to allow
+  /// inspecting the platform map state.
+  @override
+  void enableDebugInspection() {
+    GoogleMapsInspectorPlatform.instance = GoogleMapsInspectorWeb(
+        // pass this or the google maps object?
```

Comment:
I would suggest keeping `_map` private, and passing a lookup function here to replace the `GoogleMapController _map(int mapId) =>
 (GoogleMapsFlutterPlatform.instance as GoogleMapsPlugin).map(mapId)` in the inspector.

See
https://github.com/flutter/packages/blob/fba97fa4cc86a2100a06a58205a5ccb366101cdc/packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_map_inspector_ios.dart#L12-L15
and
https://github.com/flutter/packages/blob/fba97fa4cc86a2100a06a58205a5ccb366101cdc/packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_maps_flutter_ios.dart#L589-L592

Reply:
Ah yes, I like this better, good call!

Reply:
(Done)